app/components/add-cron-job.js

```diff
@@ -0,0 +1,59 @@
+import Ember from 'ember';
+import config from 'travis/config/environment';
+
+export default Ember.Component.extend({
+  classNames: ['form--cron'],
+  store: Ember.inject.service(),
+
+  reset() {
+    return this.setProperties({
+      branch: null,
+      interval: null,
+      disable_by_build: null
+    });
+  },
+
+  actions: {
+    save() {
+      var cron, self, store, repo_id, branch;
+
+      if (this.get('isSaving')) {
+        return;
+      }
+
+      self = this;
```

Comment:
A nice thing about the default Ember CLI configuration is that it comes with Babel, which lets us use things like the “fat arrow” syntax that preserves `this` in a callback. So below you could have `.then((existing_crons) { … this.set ` and be addressing the proper `this`. I see `self`s scattered around the codebase so this isn’t worth holding up a merge, but it’s something you can take advantage of in the future!